example/src/HomeScreen.tsx

```diff
@@ -99,12 +99,28 @@ function GroupListItem({
   const [messages, setMessages] = useState<
     DecodedMessage<SupportedContentTypes>[]
   >([])
+  const lastMessage = messages?.[0]
+  const [getConsentState, setConsentState] = useState<string | undefined>()
+
+  const denyGroup = async () => {
+    await client?.contacts.denyGroups([group.id])
+    group
+      .consentState()
+      .then(setConsentState)
```

Comment:
This is fine to do this way, but you are already in an async function so you could also do:
```
const consent = await group
      .consentState()
      setConsentState(consent)
```